staff/judith-gongora/app-template/about.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="home.html"><img src="https://icon-icons.com/icons2/17/PNG/256/social_html5_1970.png" alt="logo"></a>
+            <a href="profile.html">Profile</a>
+            <a href="news.html">News</a>
+            <a href="about.html">About</a>
+            <a href="contact.html">Contact</a>
+        </nav> 
+        <button type="button" onclick="location.href='login.html';">Log in</button>
+        <button type="button" onclick="location.href='register.html';">Sign in</button> 
+   </header>
+   <main>
+        <article>
+            <h1>About</h1>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit cras magna, conubia pellentesque fermentum. 
+             Purus luctus lacinia quis justo lacus sed tortor maecenas ut, pulvinar viverra rutrum dui nisi 
+             malesuada faucibus taciti, inceptos eros consequat facilisis sagittis molestie auctor neque ornare, 
+             mattis litora parturient vivamus vel integer aliquet. Venenatis pharetra mi praesent varius nam 
+             rhoncus urna taciti tempus egestas fringilla, lacinia leo habitant posuere viverra quisque 
+             pellentesque justo id vehicula, ac phasellus luctus hac netus erat suspendisse vulputate cum dis.
+            </p>
+            <p>Orci lectus suscipit facilisis auctor himenaeos eu nostra commodo feugiat, dictum maecenas sociosqu 
+             ac viverra dis vestibulum aliquam fusce, semper massa montes ligula varius litora nunc luctus. 
+             Fringilla hendrerit quis curae euismod blandit ultrices, pharetra mus sociosqu volutpat integer 
+             vel facilisi, nec dictum duis mi leo. Aptent orci auctor magna fringilla neque tortor quam molestie 
+             velit, iaculis dictumst nullam vestibulum penatibus dictum sem lacus, proin platea tincidunt vel 
+             placerat viverra blandit ultrices.
+            </p>
+            <img src="http://www.enaco.cl/img/proyectos/region_metropolitana/edificio_piramide/edificio_piramide.jpg" alt="sea">        
+        </article>
```

Comment:
may you want to use a section here, since the content inside the article is not independent from the rest of the page